### PR TITLE
Alok verma749 patch 1 update readme

### LINKED ISSUE #127: [DOCS] Make installation step 5 clearer
### Description

At the moment instruction 5 says: Add a GitHub Token in .env (Copy .env.example and paste credentials)

It would be clearer is it instead says: Create a .env file in the root of the project. Then add your GitHub token (see example in .env.example file)

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alokVerma749**
## Fixes Issue

Closes #127

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/135"></a>

### CODE REVIEW

**Comment by alokVerma749** [comment]
I updated these line, and make it more understandable.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,14 @@ Before installation, please make sure you have already installed the following t
    ```
 3. Navigate to the project directory `cd good-first-issue-finder`
 4. Install dependencies with `npm install`
-5. Add a GitHub Token in .env (Copy .env.example and paste credentials)
+5. Add secret github token in a .env file:
+
+-Add a .env file int the root directory of the folder
```
Comment (Cahllagerfeld) [inline README.md:37]:
```suggestion
-Add a `.env` file int the root directory of the folder
```

File: README.md [hunk 2]
```diff
@@ -32,7 +32,14 @@ Before installation, please make sure you have already installed the following t
...
+
+-Paste below code inside the .env file with your secret token
```
Comment (Cahllagerfeld) [inline README.md:39]:
```suggestion
-Paste below code inside the `.env` file with your secret token
```

**Review by Cahllagerfeld — COMMENTED** [review]
Adding inline-code blocks

**Comment by emmalearnscode** [comment]
This is a duplicate of another pull request and the issue is currently assigned to someone else. Please feel free to tackle an issue which isn’t currently assigned to someone.

**Comment by alokVerma749** [comment]
ok my bad

**Pull request closed without merging**